Travis build is failing

**Actual Behaviour**
* Travis build is failing
<!--Please state here what is currently happening.-->

**Expected Behaviour**
* It should pass the test
<!--State here what the feature should enable the user to do.-->

**Steps to reproduce it**
NA
<!--Add steps to reproduce bugs or add information on the place where the feature should be implemented. Add links to a sample deployment or code.-->

**Screenshots of the issue**
```
Creating an optimized production build...
Failed to compile. When process.env.CI = true, warnings are treated as failures. Most CI servers set this automatically.
/home/travis/build/fossasia/chat.susi.ai/src/components/Overview/Overview.react.js
  22:1  warning  componentWillMount is deprecated since React 16.3.0, use UNSAFE_componentWillMount instead, see https://reactjs.org/docs/react-component.html#unsafe_componentwillmount  react/no-deprecated
✖ 1 problem (0 errors, 1 warning)
/home/travis/build/fossasia/chat.susi.ai/src/components/ChatApp/Settings/Settings.react.js
  65:1  warning  componentWillMount is deprecated since React 16.3.0, use UNSAFE_componentWillMount instead, see https://reactjs.org/docs/react-component.html#unsafe_componentwillmount  react/no-deprecated
✖ 1 problem (0 errors, 1 warning)
/home/travis/build/fossasia/chat.susi.ai/src/components/ChatApp/MessageSection/MessageSection.react.js
  165:1  warning  componentWillMount is deprecated since React 16.3.0, use UNSAFE_componentWillMount instead, see https://reactjs.org/docs/react-component.html#unsafe_componentwillmount  react/no-deprecated
✖ 1 problem (0 errors, 1 warning)
/home/travis/build/fossasia/chat.susi.ai/src/components/ChatApp/MessageComposer.react.js
  61:1  warning  componentWillMount is deprecated since React 16.3.0, use UNSAFE_componentWillMount instead, see https://reactjs.org/docs/react-component.html#unsafe_componentwillmount  react/no-deprecated
✖ 1 problem (0 errors, 1 warning)
/home/travis/build/fossasia/chat.susi.ai/src/components/ChatApp/MessageListItem/VoicePlayer.js
  5:1  warning  componentWillReceiveProps is deprecated since React 16.3.0, use UNSAFE_componentWillReceiveProps instead, see https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops  react/no-deprecated
✖ 1 problem (0 errors, 1 warning)
/home/travis/build/fossasia/chat.susi.ai/src/components/ChatApp/VoiceRecognition.js
  6:1  warning  componentWillReceiveProps is deprecated since React 16.3.0, use UNSAFE_componentWillReceiveProps instead, see https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops  react/no-deprecated
✖ 1 problem (0 errors, 1 warning)
/home/travis/build/fossasia/chat.susi.ai/src/components/ChatApp/SearchField.react.js
  20:1  warning  componentWillMount is deprecated since React 16.3.0, use UNSAFE_componentWillMount instead, see https://reactjs.org/docs/react-component.html#unsafe_componentwillmount  react/no-deprecated
✖ 1 problem (0 errors, 1 warning)

```
<!--Where-ever possible attach a screenshot of the issue.-->

**Would you like to work on the issue?**
Yes
<!--Please let us know if you can work on it or the issue should be assigned to someone else.-->

--- a/.eslintrc
+++ b/.eslintrc
@@ -261,7 +261,7 @@
     // disallow the use of alert, confirm, and prompt
     "no-alert": 0,
     // disallow the use of deprecated react changes and lifecycle methods
-    "react/no-deprecated": 1,
+    "react/no-deprecated": 0,
     // disallow comparisons to null without a type-checking operator
     "no-eq-null": 0,
     // disallow overwriting functions written as function declarations

